.github/workflows/push.yml

```diff
@@ -13,8 +13,8 @@ jobs:
 
     - name: Run CI scripts
       run: |-
-        sudo make ci
+        ./scripts/ci-wrapper dev-v2.5
```

Comment:
```suggestion
        ./scripts/ci-wrapper dev-v2.5-source
```

Reply:
This actually needs to be dev-v2.5. dev-v2.5 is going to the asset branch.

Reply:
whoops, you're right. Nvm